DFC.ServiceTaxonomy.PageLocation/Validators/PageLocationUrlValidator.cs

```diff
@@ -62,6 +62,33 @@ public async Task<bool> Validate(JObject term, JObject taxonomy)
                 }
             }
 
+            IEnumerable<ContentItem> categories = await _session.Query<ContentItem, ContentItemIndex>(x => x.ContentType == "JobCategory" && x.Latest).ListAsync();
```

Comment:
This code block _could_ be factored out, as it's very similar to the block above, checking Pages.